index.bs

```diff
@@ -599,28 +543,28 @@ If there is more than one [=shape tree location=], they all apply to the same
     </thead>
     <tbody>
       <tr>
-        <td><code class="vocab">st:hasShapeTree</code></td>
+        <td><code class="vocab">st:assigns</code></td>
```

Comment:
Note that https://github.com/xformativ/shapetrees-java/pull/108#discussion_r742755985 proposes `testsConformanceWith`